tiptapy/__init__.py

```diff
@@ -102,20 +102,31 @@ class Image(BaseNode):
 
     def is_renderable(self, node):
         attrs = node.get("attrs", {})
-        return bool(attrs.get('src', '').strip())
+        src = attrs.get("src", "")
+        if src:    
+            src =bool(
+                src.get('image', '').strip() and src.get('fallback', '').strip()
+            )
+        return src
```

Comment:
@Pradhvan It should return T/F
So simple `return src and bool(...)` should work.